File: projectC/awale/src/ai/evaluation1.c
#include "../../headers/ai/evaluation1.h"

int evaluate_evaluation1(Player player, int nbSeedCapturedByMe, int nbSeedCapturedByOpponent, int board[16][3])
{
    int diffNbSeedCaptured = nbSeedCapturedByMe - nbSeedCapturedByOpponent;
    int diffMobility = mobility(&(player), board) - mobility(player.opponent, board);

    // Others variables (to complete)

    return diffNbSeedCaptured * 2 + diffMobility;
}

File: project/src/main/java/awale/ai/Evaluation1.java
package awale.ai;

import awale.game.Bot;

public class Evaluation1 implements Evaluation {

	@Override
	public int evaluate(Bot bot, int nbSeedCapturedByMe, int nbSeedCapturedByOpponent, int[][] board) {
		int diffNbSeedCaptured = nbSeedCapturedByMe - nbSeedCapturedByOpponent;
		int diffMobility = bot.mobility(board) - bot.getOpponent().mobility(board);

		// others variables (to complete)

		return diffNbSeedCaptured * 2 + diffMobility;
	}

}
